content/post/performance-metrics-histogram.md

```diff
@@ -0,0 +1,245 @@
++++
+title = "Latency Histograms and Percentile Distributions In Wallaroo Performance Metrics"
+date = 2018-02-15T07:11:00-05:00
+draft = false
+author = "nisanharamati"
+description = "How we implemented Wallaroo's low overhead performance counters and the philosophy behind the choices we made."
+tags = [
+    "performance",
+    "metrics",
+    "monitoring"
+]
+categories = [
+    "Observability"
+]
++++
```

Comment:
this is missing a `slug`, can you add one so I know how to reference to this post from mine

Reply:
We do not use slug. There shouldn't be a slug.

Reply:
We don't actually use it. It's misleading.

Reply:
The url from the hosting domain or what you get locally is what you want.

Its based on the date, and the title of the post.

except you want it as https://blog.wallaroolabs.com instead of a netlify-preview or localhost.